snappy/services.go

```diff
@@ -322,15 +277,15 @@ WantedBy={{.ServiceSystemdTarget}}
 	return templateOut.String()
 }
 
-func GenSocketFile(desc *ServiceDescription) string {
+func GenSocketFile(appInfo *snap.AppInfo) string {
 	serviceTemplate := `[Unit]
-Description={{.Description}} Socket Unit File
+Description=socket {{.App.Name}} for snap {{.App.Snap.Name}} - autogenerated DO NO EDIT
```

Comment:
Hmm.. that will be a weird sentence depending on the name of the snap and app.

How about:

`Socket for snap application {{.App.Snap.Name}}.{{.App.Name}}`